store/src/main/java/com/nytimes/android/external/store/base/impl/MemoryPolicy.java

```diff
@@ -0,0 +1,62 @@
+package com.nytimes.android.external.store.base.impl;
+
+
+import java.util.concurrent.TimeUnit;
+
+public class MemoryPolicy {
```

Comment:
Since this is a public class, a javadoc would be appreciated. Could you write class/method comments where appropriate?